web/src/context/Population.svelte

```diff
@@ -27,7 +27,16 @@
   let showCarOwnership = false;
 </script>
 
-<ContextLayerButton label="SIMD" bind:show={showSIMD}>
+<ContextLayerButton
+  label="SIMD"
+  bind:show={showSIMD}
+  onChange={() => {
```

Comment:
One slightly disorienting thing is the position of the clicked button changing when the one above gets turned off. Slightly hard to see without the mouse showing up (ubuntu is still broken there)

[Screencast from 23-04-25 10:06:12.webm](https://github.com/user-attachments/assets/61ffad63-bb23-472f-b106-6424c1634540)

Reply:
Yeah, I see what you mean. It is a little disorienting, but I think the upside outweighs the downside.